runtime/compiler/runtime/JitRuntime.cpp

```diff
@@ -1106,6 +1106,30 @@ extern "C" void _patchJNICallSite(J9Method *method, uint8_t *pc, uint8_t *newAdd
    // simply patch in the new address
    *(uint8_t**)pc = newAddress;
    }
+#elif defined(TR_HOST_ARM64)
+extern "C" void _patchJNICallSite(J9Method *method, uint8_t *pc, uint8_t *newAddress, TR_FrontEnd *fe, int32_t smpFlag)
+   {
+   // We expect the following code sequence:
+   // movzx scratchReg, imm16
+   // movkx scratchReg, imm16, LSL16
+   // movkx scratchReg, imm16, LSL32
+   // movkx scratchReg, imm16, LSL48
+   uintptr_t address = reinterpret_cast<uintptr_t>(newAddress);
+   uint32_t *cursor = reinterpret_cast<uint32_t*>(pc);
+
+   TR_ASSERT((*cursor & 0xffe00000) == 0xd2800000, "unexpected JNI call sequence at %p: %p", cursor, *cursor);
```

Comment:
The type of `*cursor` is `uint32_t`.  So the message in `TR_ASSERT()` should be "unexpected JNI call sequence at %p: %08x" or something like it.

Reply:
Thanks for pointing it out. Updated the format string.